website/source/docs/guides/production-acls.html.md

```diff
@@ -155,49 +160,35 @@ Repeat this process for each agent. It is the responsibility of the operator to
 save tokens in a secure location; we recommend
 [Vault](https://www.vaultproject.io/).  
 
-### Add the token to the Agent.
+### Add the Token to the Agent.
 
-Finally, apply the tokens to the agents by adding the token to the agent
-configuration file and restarting the Consul service. Start with the servers
-and ensure they are working correctly before applying the client tokens. Below
-is an example server configuration file with an agent token.
+Finally, apply the tokens to the agents using the HTTP API. 
+Start with the servers
+and ensure they are working correctly before applying the client tokens. Please
+review the Bootstrapping the ACL System [guide](/advanced/day-1-operations/acl-guide) for example of setting the token in the agent configuration 
+file.
 
-````
-# consul-server-one.hcl
-{ 
-"acl": { 
-  "enabled": true, 
-  "default_policy": "deny", 
-  "tokens": { 
-  	"agent": "e67a4a33-6579-b766-ac97-a0408fde3957" } 
-  }
-} 
-```` 
+```sh
+$ curl -H "X-Consul-Token: <your token here>" -D @consul-server-one-token.json http://localhost:8500/v1/agent/token/acl_agent_token 
+``` 
+
+The data file must contain a valid token. 
```

Comment:
If you take the previous change to use the Consul CLI instead of curl then this bit about the data file can be removed. Totally up to you.

Reply:
Thanks for pointing that out! To be consistent I will change it to CLI.